### DISCUSSION TITLE
Trigger action when team scores

### DISCUSSION THREAD

**Discussion by RedDoughnut** [vasqued2/ha-teamtracker#113] [General]
Is this doable? If not could you add this?

Thanks

**Comment by NRutt2005** [comment]
This is doable. In fact, I've been using this for quite sometime in Home Assistant. 
Here is a link to one of the automations I have setup through this custom integration. When the Texas Rangers baseball team scores it runs a  script I setup to change from LED in my house to a certain color, but only if we're home and then after 1 minute it runs some checks to determine what setting the lights should return to.

Automation for the team scores:
[https://pastebin.com/XDLz7AL9](url)

Script:
[https://pastebin.com/igrBw6xy](url)

Hope this helps.

Note: You will notice I am also getting a push notification to my phone from the Home Assistant App when the automation is ran. This push notification gives me the Inning and the score for both teams. There is room for improvement on my automation using trigger IDs but for now, this is what I have.

**Reply by sbailey151** [reply]
HI @NRutt2005 .  Do you have new links you can post?  These links seem to be broken.

**Comment by vasqued2** [comment]
[https://github.com/vasqued2/ha-teamtracker/wiki/Automation-Examples](https://github.com/vasqued2/ha-teamtracker/wiki/Automation-Examples)
